src/app/common/mongoose/paging-search.plugin.js

```diff
@@ -2,6 +2,8 @@ const
 	deps = require('../../../dependencies'),
 	config = deps.config;
 
+const MONGO_TIMEOUT_ERROR_CODE = 50;
```

Comment:
Couldn't find if Mongo/mongoose had an error type for timeout, so I guess 50 it is?